.changesets/use-send_session_data-instead-of-skip_session_data.md

```diff
@@ -0,0 +1,6 @@
+---
+bump: "patch"
+type: "change"
+---
+
+The `skip_session_data` config option is now deprecated in favor of `send_session_data`.
```

Comment:
```suggestion
Update the plug integration to listen to the `send_session_data` config option, instead of `skip_session_data`. The `send_session_data` config option is backwards compatible with the `skip_session_data` config option.
```